apps/betterangels/eas.json

```diff
@@ -43,7 +43,7 @@
       "ios": {
         "ascAppId": "6468135540"
       },
-     "android": {
+      "android": {
```

Comment:
why is this file updated?

Reply:
haven't touched. everything was merged from main branch. and accepted all incoming.